packages/wp-now/README-new.md

```diff
@@ -0,0 +1,140 @@
+# wp-now
+
+`wp-now` streamlines the process of setting up a local WordPress environment.
+
+It uses automatic mode detection to provide a fast setup process, regardless of whether you're working on a plugin or an entire site. You can easily switch between PHP and WordPress versions with just a configuration argument. Under the hood, `wp-now` is powered by WordPress Playground and only requires Node.js.
+
+## Getting Started
+
+To install `wp-now` directly from `npm`:
+
+```bash
+npm install -g @wp-now/wp-now
+```
+
+Alternatively, you can install `wp-now` via `git clone` if you'd like to hack on it too. See [Contributing](#contributing) for more details.
+
+Once installed, you can use it like so:
+
+```bash
```

Comment:
`wp-now start` will work if we install it globally via `npm install -g @wp-now/wp-now`.

`npm link` should be used to make `wp-now` available globally if we use the project in development mode: for example, when we clone the repository to review and test changes from the branch.

Reply:
> I think we need to adjust these instructions because `wp-now start` is going to work only if you run `npm link` after cloning `wp-now` from GitHub.

There's a certain assumption of knowledge, too:
* If you're a _user_, simply run `npm install -g @wp-now/wp-now` and you're good to go.
* If you want to clone the project repo and contribute, you'll need some additional knowledge to be successful.

We can further clarify the **Contributing** section as we go along, but `git clone` usage should be documented separately from standard usage.